sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -27,17 +27,35 @@ point of entry for receiving of any type (from single partition, all partitions,
 | `EventHubClient.createFromAadTokenCredentials()` | `new EventHubProducerClient()` or `new EventHubConsumerClient()` | [usingAadAuth](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/usingAadAuth.ts)
 | `EventProcessorHost.createFromConnectionString()`                           | `new EventHubConsumerClient(..., checkpointStore)`               | [receiveEventsUsingCheckpointStore](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/receiveEventsUsingCheckpointStore.ts) |
 
+Other noteworthy changes:
+- In v5, the `EventHubConsumerClient` class takes the consumer group name as a mandatory argument in its constructor.
+If you havent created any consumer groups explicitly, then use the name of the default consumer group which is `$Default`.
```

Comment:
Should we mention `EventHubConsumerClient.defaultConsumerGroupName`?

Reply:
I was under the impression we were trying to NOT mention it explicitly.

Reply:
I would if the default group name would change, but it wont.
The service cant do that without it being a breaking change.
So, its simpler to use the literal value than introduce a re-direct.